std/net/isemail.d

```diff
@@ -75,6 +71,11 @@ if (isSomeChar!(Char))
     import std.uni : isNumber;
 
     alias tstring = const(Char)[];
+    alias TstringToken = Token!(Char);
```

Comment:
A simpler change is to rename current Token --> TokenImpl, and then add `alias Token = TokenImpl!Char`